packages/fxa-auth-server/lib/oauthdb/grant-tokens-from-authorization-code.js

```diff
@@ -33,6 +33,7 @@ module.exports = config => {
         access_token: validators.accessToken.required(),
         refresh_token: validators.refreshToken.optional(),
         id_token: validators.assertion.optional(),
+        session_token: Joi.string().optional(),
```

Comment:
IIUC, this is the point where you want to return the full session token rather than just its id, so I think changing the names of the others will help make it more obvious that something special is happening here.

Reply:
we should chat about this in the 1:1